CTLD.lua

```diff
@@ -4069,8 +4435,8 @@ function ctld.unpackMultiCrate(_heli, _nearestCrate, _nearbyCrates)
         end
 
 
-        local _spawnedGroup = ctld.spawnCrateGroup(_heli, { _point }, { _nearestCrate.details.unit })
-
+        local _spawnedGroup = ctld.spawnCrateGroup(_heli, { _point }, { _nearestCrate.details.unit }, { _crateHdg })
+        ctld.setGrpROE(_spawnedGroup)
```

Comment:
This function is generating an error, not clear on the purpose of this, and only in unpackMultiCrate.
CTLD ERROR: [string "ctld.lua"]:6510: Parameter #1 (group name) missed

Reply:
I added an issue and am clearing this comment